test/ILLink.RoslynAnalyzer.Tests/DataFlowTests.cs

```diff
@@ -73,10 +73,16 @@ public Task FieldDataFlow ()
 			return RunTest (nameof (FieldDataFlow));
 		}
 
+		[Fact]
+		public Task GenericParameterDataFlowWithoutMakeGeneric ()
```

Comment:
nit: I would just call this `GenericParameterDataFlow`, and the other one something like `MakeGenericDataFlow`